src/Core/TempFolder.php

```diff
@@ -18,9 +18,15 @@ class TempFolder
     public static function getTempFolder($base)
     {
         $parent = static::getTempParentFolder($base);
+        $user = static::getTempFolderUsername();
```

Comment:
```suggestion
        $folderName = static::getTempFolderUsername();
```
It's not just the user anymore